src/app/modules/login/components/login-form/login-form.component.ts

```diff
@@ -10,13 +11,15 @@ import { BehaviorSubject } from 'rxjs';
 })
 export class LoginFormComponent implements OnInit{
   @Output() loginEvent = new EventEmitter<UserLogin>();
+  @Output() loginGoogleEvent = new EventEmitter<string>();
 
   @Input() errorForm: BehaviorSubject<boolean>;
   @Input() loading: BehaviorSubject<boolean>;
+  @Input() loadingGoogle: BehaviorSubject<boolean>;
```

Comment:
checking whether login process is in progress is not responsible of LoginFormComponent ? Always ask yourself "what's responsible of the function/class/component..." to make sure that they do right things they should

Reply:
Remove all inputs